### DISCUSSION TITLE
Why the need to "sign in"

### DISCUSSION THREAD

**Discussion by shuntera** [open-webui/open-webui#491] [General]
I don't need to sign in to use ollama, so why the need to sign in to use the webui? Does the login stay local or are you harvesting email addresses?

**Comment by justinh-rahb** [comment]
Login is NOT Required:
https://docs.openwebui.com/getting-started/env-configuration#webui_auth


---

The login is strictly local, no emails or other information are being harvested. The login is required to protect your Ollama and external APIs from unauthorized access. If you want to run a version without authentication, there is ollama-webui-lite that's designed to work without a backend (direct browser client -> Ollama API).

**Reply by neilyoung** [reply]
But what is required to login? I'm always confronted with a sign in page, but I can't login, since my credentials are lost. How can I recover from this?

**Reply by xutianyi** [reply]
same here

**Reply by xutianyi** [reply]
I just entered a random email address and password then it let me in

**Comment by Coffee-Boyy** [comment]
I have a feeling this will be remediated with the introduction of a dedicated application: https://github.com/open-webui/open-webui/issues/8262
